api/src/test/resources/io/strimzi/api/kafka/model/Kafka-with-missing-required-property.yaml

```diff
@@ -3,7 +3,9 @@ kind: Kafka
 metadata:
   name: missing-required-property
 spec:
-  topicOperator:
```

Comment:
Same here, if I don't fix the yaml, the test won't pass with the expected behavior.

Reply:
Yeah, I think you need to fix the YAML here.